lightning/src/ln/functional_test_utils.rs

```diff
@@ -97,36 +99,6 @@ impl<'a, 'b, 'c> Drop for Node<'a, 'b, 'c> {
 			assert!(self.node.get_and_clear_pending_events().is_empty());
 			assert!(self.chan_monitor.added_monitors.lock().unwrap().is_empty());
 
-			// Check that if we serialize the Router, we can deserialize it again.
```

Comment:
Why did this get removed?

Reply:
Network graph serialization is already tested [here](https://github.com/naumenkogs/rust-lightning/blob/97ac065418832b7e0e42de246d7606241a235ae4/lightning/src/routing/network_graph.rs#L1584), unless you mean something else.

Reply:
This check is a bit different - in the past we've had issues in edge cases failing to serialize-deserialize properly. So we added this so that after every test (which often test fun edge cases), we can serialize-deserialize everything properly. I think we should keep this, at least for the underlying graph.